scripts/update/Updater.php

```diff
@@ -951,5 +951,12 @@ public function update($initialVersion)
         }
 
         $this->skip('19.10.0', '19.14.0');
+
+        if ($this->isVersion('19.14.0')) {
+            $script = 'sudo -u www-data php index.php \'oat\taoProctoring\scripts\tools\KvMonitoringMigration\' -f item_duration,stored_item_duration -d 1 -s 0 -pc -l 32';
+            $this->addReport(\common_report_Report::createInfo("Run script :'" . $script . "' to finish updating. Time to completion depends on table size."));
```

Comment:
Please add a not that after launch of that script config files will be automatically changed and must be synchronized across all the instances.

Reply:
During execution of `KvMonitoringMigration` script there is a message about config synchronization: `Config persisted (ONLY ON THE CURRENT SERVER. UPDATE CONFIGS ON ALL OTHER SERVERS)`.